index.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Alick's Portfolio</title>
+  <meta name="Alick Nyirenda" content="Alick Portfolio">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <p class="header-text">Alick Nyirenda </p>
+    <a href="#"><i class="header-icon fa fa-bars fa-lg"></i></a>
+  </header>
+  <main>
+  <section class="headline">
+    <h1 class="headline-text-1">Alick Nyirenda</h1>
+    <p class="headline-text-2">Software Developer</p>
```

Comment:
- [ ] As this is a sub-header, you should also use a header `hx` tag.
_See my comments above about semantic HTML tags_

Reply:
:+1: Resolved :heavy_check_mark: